### PR TITLE
Add Step by step fed stats tabular example

### PR DESCRIPTION
**Pull request by chesterxgchen**
### Description
This PR add federated stats example using Higgs tabular dataset, This is the 1st example in the step-by-step series for Tabular data with Higgs dataset

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb [hunk 1]
```diff
@@ -0,0 +1,918 @@
...
+    "# Tabular Data Federated Statistics \n",
+    "\n",
+    "Before we perform machine learning tasks on the Higgs data, let's examine the statistics of the dataset. \n",
+    "\n",
+    "\n",
+    "## Setup NVFLARE\n",
+    "\n",
+    "Follow [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html) to set up a virtual environment and install NVFLARE.\n",
+    "\n",
+    "You can also follow this [notebook](../../nvflare_setup.ipynb) to get set up.\n",
```
Comment (SYangster) [inline examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb:17]: need to update link for new folder structure (same in cifar10/image_stats.ipynb)

File: examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb [hunk 2]
```diff
@@ -0,0 +1,918 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6c8e59bd-0b97-4ad9-a0de-abd10dd76d57",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "cd code"
```
Comment (SYangster) [inline examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb:444]: add "!"

File: examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb [hunk 3]
```diff
@@ -0,0 +1,918 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b0d8014-ed54-4d0e-bc96-adad905a29dd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "cd ../."
```
Comment (SYangster) [inline examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb:565]: add "!"

**Review by SYangster — COMMENTED** [review]
Good step-by-step example, minor comments

File: examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb [hunk 4]
```diff
@@ -0,0 +1,918 @@
...
+   "id": "d19adbde-b204-4483-810b-56c5d1517112",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Create Federated Statistics Job\n",
+    "\n",
+    "We are going to use NVFLARE job cli to create job. For detailed instructions on Job CLI, please follow the [job cli tutorial](https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/job_cli.ipynb)\n",
+    "\n",
+    "Let's check the available job templates, we are going to use one of the existing job template and modify to fit our needs. The job template is nothing but server and client-side job configurations."
```
Comment (YuanTingHsieh) [inline examples/hello-world/step-by-step/higgs/stats/tabular_stats.ipynb:579]: ``job template**s** and modify **it** to fit our needs.``

**Review by YuanTingHsieh — COMMENTED** [review]
Contents are good.
Suggest a few grammar fixes.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks LGTM

**Pull request merged**